.github/ISSUE_TEMPLATE/issue-with-custom-model.md

```diff
@@ -0,0 +1,65 @@
+---
+name: Issue with custom model
+about: Ask for help if you have issue with your model
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+> **Please do not disclose security vulnerabilities as issues. See our [security policy](../../SECURITY.md) for responsible disclosures.**
+
+###Before opening an issue
+**Issue Checklist**
+
+- [ ] I have checked the [compatibility table](https://axiscommunications.github.io/acap-documentation/docs/api/computer-vision-sdk-apis.html#compatibility) in the documentation.
+- [ ] I have verified that the SDK version I am using is compatible with the firmware version installed on my device.
+- [ ] I have included the SDK version and firmware version in my issue description for reference.
+- [ ] I have followed the minimum debug steps on my own.
+- [ ] I have included the output of the debug steps in the issue.
+
+**Minimum Debug Steps**
+Before opening the issue, try to load the model on the device using this command:
+`larod-client -g model_path  -c <CHIP_ID> `
+
+Where <CHIP_ID> is:
+'cpu-tflite' - To test on CPU
+'axis-a8-dlpu-tflite' - To test on Artpec-8 DLPU
+'google-edge-tpu-tflite' - To test on Artpec-7 TPU
+'ambarella-cvflow'- To test on CV25 DLPU
+    
+If it fails run
+`journalctl -u larod`
+Please provide us this log together with your issue.
```

Comment:
``````suggestion
If it fails run:

```
journalctl -u larod
```

Please provide us this log together with your issue.
``````